Launcher/Storyboards/Base.lproj/Main.storyboard

```diff
@@ -75,11 +75,16 @@
                                 <modifierMask key="keyEquivalentModifierMask"/>
                                 <menu key="submenu" title="Settings" id="eMl-Tr-UV4">
                                     <items>
-                                        <menuItem title="Configure Settings" keyEquivalent="s" id="QJc-YU-Tln">
+                                        <menuItem title="Configure settings" keyEquivalent="s" id="QJc-YU-Tln">
```

Comment:
Let's keep this uppercased as per the [Human Interface Guidelines](https://developer.apple.com/ios/human-interface-guidelines/controls/buttons/):

> Use title-case for titles. Capitalize every word except articles, coordinating conjunctions, and prepositions of four or fewer letters.